portal/src/views/viz/SharePanel.vue

```diff
@@ -118,6 +124,13 @@ export default Vue.extend({
                 }, 3000);
             });
         },
+      openMailClient(): void {
+            const body = this.getCurrentURL();
+            const subject = isCustomisationEnabled()
+                ? this.$t("sharePanel.emailSubject.floodnet")
+                : this.$t("sharePanel.emailSubject.fieldkit");
+            window.location.href = "mailto:?subject=" + subject + "&body=" + body;
+        },
```

Comment:
One thing I'd like to do more of is move this conditional logic to the "active" PartnerCustomizations data. I also think we should have a "default" PartnerCustomizations that's filled with default values.

So, in getPartnerCustomization we return a default instead of null, and we can add a "email": { "subject": "sharePanel.emailSubject.fieldkit" } to the "sharing" block, and then use the floodnet key in theirs. That way there's no conditional here in sharing nor is there a reference to floodnet or any other future partner.

Reply:
I am not sure how to do this without a lot of refactoring throughout the project.
isCustomisationEnabled is used everywhere. If I add a default (for fieldkit) PartnerCustomization then this function will not return true anymore. I don't see how to make this change with impact only on the share panel.